src/properties.c

```diff
@@ -217,20 +218,20 @@ static int set_data_properties(struct knot_thing *thing,
 
 	storage_read_key_int(fd, group_id, DATA_IP_ELEMENT_SIZE,
 			&element_size_aux);
-	if (element_size_aux < 0)
+	if (element_size_aux == -EINVAL)
 		return -EINVAL;
 
 	data_item_aux.element_size = element_size_aux;
 
 	storage_read_key_int(fd, group_id, DATA_REG_ADDRESS,
```

Comment:
what does fd means?

Reply:
File destination. It is a common way of abbreviating.

Reply:
File destination. It is a common way of abbreviating.